routers/repo/webhook.go

```diff
@@ -646,7 +650,7 @@ func checkWebhook(ctx *context.Context) (*orgRepoCtx, *models.Webhook) {
 		return nil, nil
 	}
 
-	ctx.Data["HookType"] = w.Type
+	ctx.Data["HookType"] = strings.TrimSpace(w.Type)
```

Comment:
This is currently needed for webhook edit to even work because char(x) in database adds spaces. This should be removed if PR #14148 is merged before this or in that PR if this PR is merged as first

Reply:
Most likely should not go ahead with this change, seeing as per-repo webhook form has the same issue and it's not fixed there.

Reply:
I can revert it, I just added it here as otherwise I could not test my changes, I actually thought I have broken something and lost about 30 minutes on thinking what I have done wrong :D

Reply:
I'd say revert; #14148 is blocking anyway as it's kind of critical so 1.14 isn't happening without it.

Reply:
Removed this change, so this PR keeps it as broken as it was 🤣